src/contexts/SiteLaunchContext.tsx

```diff
@@ -68,6 +97,27 @@ export const SiteLaunchProvider = ({
      */
     queryClient.invalidateQueries([SITE_DASHBOARD_LAUNCH_STATUS_KEY, siteName])
   }
+
+  const [pageNumber, setPageNumber] = useState<SiteLaunchPageIndex>(
+    getInitialPageNumber(siteLaunchStatusProps)
+  )
+
+  const increasePageNumber = () => {
+    if (pageNumber === SITE_LAUNCH_PAGES.FINAL_STATE) {
```

Comment:
nit: am okay with this, but to be cautious I usually do a `>=` check and reset it back to first or last index to account for the very edge case where we accidentially cross the boundaries

Reply:
fair 19b5ab15e121ec44aff3fb9e854bcb470678b321